tools/rapids-download-conda-from-github

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# A utility script that downloads a conda artifact archive from Github Artifacts, unzips it,
+# and returns the path to the directory where it was unzipped.
+# Positional Arguments:
+#   1) a string of "cpp" or "python" which determines which conda artifact
+#      should be downloaded
+set -euo pipefail
+export RAPIDS_SCRIPT_NAME="rapids-download-conda-from-github"
+
+pkg_type="$1"
+# Validate package type argument
+if [[ "${pkg_type}" != "cpp" && "${pkg_type}" != "python" ]]; then
+  rapids-echo-stderr 'Pass "cpp" or "python" as an argument.'
+  exit 1
+fi
```

Comment:
Can we make this a `case` statement for consistency with some of the existing scripts?